### PR TITLE
8.2 ipv6

### PR DESCRIPTION
**Pull request by benjamreis**

### CODE REVIEW

File: SPECS/xapi.spec [hunk 1]
```diff
@@ -465,6 +474,18 @@ Coverage files from unit tests
 %endif
 
 %changelog
+* Fri Jan 29 2021 Benjamin Reis <benjamin.reis@vates.fr> - 1.249.5-.1.1.0.ipv6.1
```
Comment (stormi) [inline SPECS/xapi.spec:477]: typo in version number

Comment (benjamreis) [reply SPECS/xapi.spec]: Good catch! Solved

File: SPECS/xapi.spec [hunk 2]
```diff
@@ -1028,7 +1049,7 @@ Coverage files from unit tests
 - CA-258385: Improved phrasing for errors thrown by assert_can_migrate.
 - Replace /tmp/network-reset literal with Xapi_globs
 - Travis: remove opam-coverage
-- Modifications to the error messages for better compliance with the 
+- Modifications to the error messages for better compliance with the
```
Comment (stormi) [inline SPECS/xapi.spec:1052]: Unexpected change (trailing space removed). Many more below.

Comment (benjamreis) [reply SPECS/xapi.spec:1052]: I saw but since it's not designed to be merged i thought it was okay to leaeve it.

Comment (stormi) [reply SPECS/xapi.spec:1052]: Ok

**Comment by stormi** [comment]
Looks good packaging-wise.

**Pull request closed without merging**